app/templates/events/event_list.html

```diff
@@ -30,18 +30,18 @@ <h1 class="text-center mt-5">Events List for {{selectedTerm.description}}</h1>
     </div>
   </div>
 </div>
-<div>
+<div class="row">
   <ul class="nav nav-tabs nav-fill mx-3 mb-3" id="pills-tab" role="tablist">
-  <li class="nav-item" role="presentation">
+  <li class="col-md-3 col-sm-12 col-12 nav-item" role="presentation">
     <button class="nav-link active" id="studentLedEvents"
     data-bs-toggle="pill" data-bs-target="#pills-student-led" type="button" role="tab" aria-controls="pills-student-led" aria-selected="true">Student Led Service</button>
   </li>
-  <li class="nav-item" role="presentation">
+  <li class="col-md-3 col-sm-12 col-12 nav-item" role="presentation">
     <button class="nav-link" id="trainingEvents"
     data-bs-toggle="pill" data-bs-target="#pills-training" type="button" role="tab" aria-controls="pills-training" aria-selected="false">Training and Education</button>
   </li>
   {% if user.isCeltsAdmin or user.isCeltsStudentStaff or user.isBonnerScholar %}
-    <li class="nav-item" role="presentation">
+    <li class="col-md-3 col-sm-12 col-12 nav-item" role="presentation">
```

Comment:
The `col-sm-12` class is redundant here, because it's already size 12 (from `col-12`). It goes from smallest to largest, and you need the class where the breakpoint is. In this case, at medium.